src/FeedbackBot/Controllers/HomeController.cs

```diff
@@ -11,14 +11,28 @@ namespace FeedbackBot.Controllers
 {
     [Authorize(ActiveAuthenticationSchemes = "ucdcas")]
     public class HomeController : Controller
-    {  
-        public async Task<IActionResult> Index()
+    {
+
+        public static class Globals
+        {
+            public static String kerberos { get; set; } 
+        }
+
+        public GitHubClient initialize()
         {
-            // Github Authentication
             var client = new GitHubClient(new ProductHeaderValue("FeedbackBot"));
-            var basicAuth = new Credentials("UCDFeedbackBot", "99LinesOfCode"); 
+            var basicAuth = new Credentials("UCDFeedbackBot", "99LinesOfCode");
             client.Credentials = basicAuth;
-            var kerberos = User.Identity.Name;
+            return client;
+        }
+
+        public async Task<IActionResult> Index()
+        {
+            // Github Authentication
+            var client = initialize();
```

Comment:
@cszhu Since you are initializing this everywhere I think you can change it to just init in the constructor and expose it as a field and each method will have access then to the initialized client.  `_client = initialize()`